sandy/core/cov.py

```diff
@@ -399,14 +406,39 @@ def glls(self, S, Vy):
         C = Vx - Vx.dot(S.dot(np.linalg.inv(V).dot(S.T.dot(Vx))))
         return self.__class__(C)
 
-    def sandwich(self, Si, Sj=None):
+    def sandwich(self, S):
+        """
+        Apply the sandwich formula to the CategoryCov object for a given
+        pandas.Series.
+
+        Parameters
+        ----------
+        S : `pd.Series`
+            Standard desviations.
+
+        Returns
+        -------
+        `CategoryCov`
+            `CategoryCov` object to which we have applied sandwich
+            formula for a given pd.Series
+
+        Examples
+        --------
+        >>> S = np.diag(np.array([1,2,3]))
+        >>> var = pd.Series([1, 2, 3])
+        >>> cov = sandy.CategoryCov(S)
+        >>> cov.sandwich(var)
+                    0           1           2
+        0 1.00000e+00 0.00000e+00 0.00000e+00
+        1 0.00000e+00 8.00000e+00 0.00000e+00
+        2 0.00000e+00 0.00000e+00 2.70000e+01
+        """
+        if self.data.index.values.all() != S.index.values.all():

```

Comment:
why don't you try [`pandas.Index.equals`](https://pandas.pydata.org/docs/reference/api/pandas.Index.equals.html)?

Reply:
When not entered in the index as such, this function breaks. By this I mean that with indexes like pd.Index([1, 2, 3]) it works, but in the example the index is not specified [RangeIndex(start=0, stop=3, step=1)], and in this case it does not work. 

However, I'll add a test to check the case pd.Index([1, 2, 3])

Reply:
maybe we should remove this...again we expect that the users know what they are doing.
just specify in the docstring!

Reply:
This is still here